vpr/src/power/power_sizing.cpp

```diff
@@ -379,7 +379,7 @@ static double power_count_transistors_switchbox() {
 
     auto& device_ctx = g_vpr_ctx.device();
     const auto& rr_graph = device_ctx.rr_graph;
-    for (size_t seg_idx = 0; seg_idx < device_ctx.rr_segments.size(); seg_idx++) {
+    for (size_t seg_idx = 0; seg_idx < device_ctx.rr_graph_builder.num_rr_segments(); seg_idx++) {
```

Comment:
Consider to use ``RRGraphView`` here and move ``num_rr_segments()`` to ``RRGraphView``.